book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] As the reviewer addressed before, please pass the book as a parameter to a new rental object. So you need to create a rental here. Pass (person, date, self) as an argument to create a rental. And add that rental to the book's rentals with @rentals << rental